app/enrichment/validate-header.js

```diff
@@ -3,7 +3,9 @@ const schema = require('./schemas/header')
 const validateHeader = (header) => {
   const validationResult = schema.validate(header)
   if (validationResult.error) {
-    throw new Error(`Header is invalid. ${validationResult.error.message}`)
+    const error = new Error('Header is invalid', { cause: validationResult.error })
+    error.category = 'validation'
```

Comment:
Could this be a lookup to avoid misspelling?

Reply:
Agreed and updated